build-tools/scripts/RunTests.targets

```diff
@@ -175,6 +175,7 @@
   <ItemGroup>
     <_RunTestTarget Include="RunJavaInteropTests" />
     <_RunTestTarget Include="RunPerformanceTests" />
+    <_RunTestTarget Include="RunPerformanceApkTests" />
```

Comment:
This is not-quite-right.  `RunTests.targets` doesn't `<Import/>` the `tests/RunApkTests.targets` file, and thus targets within `tests/RunApkTests.targets` are not available.  The targets mentioned in this item group are targets within *this* file, within `build-tools/scripts/RunTests.targets`.

To make this work, *this* file needs to *add* a new `RunPerformanceApkTests` target:

```xml
  <Target Name="RunPerformanceApkTests">
    <MSBuild 
        ContinueOnError="ErrorAndContinue"
        Projects="$(_TopDir)\tests\RunApkTests.targets"
        Targets="RunPerformanceApkTests"
    />
  </Target>
```

Reply:
Indeed. I didn't notice the `RunTests.targets` calls itself.

I will try to make `RunTests/RunApkTests` target handle both, just calling different targets in `RunApkTests.target` as we still need all the flavors.